ckanext/stadtzhtheme/descr.yaml

```diff
@@ -1,11 +1,10 @@
 zip:
-    description: Shapefile. Projektion CH1903+ / LV95 (EPSG:2056).
-    link: Weitere Informationen zu Shapefiles finden Sie in unserer Rubrik Werkstatt unter [Informationen zu Datenformaten.](https://www.stadt-zuerich.ch/portal/de/index/ogd/werkstatt/shp_shapefile.html)
```

Comment:
It seems that the link was removed unintentionally?

Reply:
Ok, it was intentional, but without knowing the consequences ;)

Reply:
thanks for changing it :)